esi_leap/tests/api/controllers/v1/test_offer.py

```diff
@@ -84,12 +97,16 @@ def test_one(self):
         data = self.get_json('/offers')
         self.assertEqual(self.test_offer.uuid, data['offers'][0]["uuid"])
 
+    @mock.patch('esi_leap.api.controllers.v1.utils.ro_factory.'
+                'ResourceObjectFactory.get_resource_object')
     @mock.patch('oslo_utils.uuidutils.generate_uuid')
     @mock.patch('esi_leap.api.controllers.v1.utils.check_resource_admin')
     @mock.patch('esi_leap.objects.offer.Offer.create')
     @mock.patch('esi_leap.api.controllers.v1.offer.' +
                 'OffersController._add_offer_availabilities')
-    def test_post(self, mock_aoa, mock_create, mock_cra, mock_generate_uuid):
+    def test_post(self, mock_aoa, mock_create, mock_cra, mock_generate_uuid,
```

Comment:
I wonder do we need tests on post/get with project_name? Seems like test_keystone is enough but I'm not sure.

Reply:
Good question! For unit testing purposes, it's enough to have test_keystone, as test_keystone ensures that the project-name-or-uuid functionality works, while test_offer just ensures that we call the keystone function.

Reply:
Sounds fair.